### PR TITLE
Issue 87 - Hadoop InputFormat tests should check property values

### PR DESCRIPTION
**Pull request by nidmgh**
Signed-off-by: Demai Ni <nidmgg@gmail.com>

Thank you for contributing to JanusGraph.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you written and/or updated unit tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [x] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [x] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by nidmgh** [comment]
issue 87. first PR. thanks for reviewing

**Comment by jerryjch** [comment]
Fixes #87

File: janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java [hunk 1]
```diff
@@ -70,12 +70,18 @@ public void testReadWideVertexWithManyProperties() throws Exception {
         Map<String, Object> propertiesOnVertex = graph.traversal().V().valueMap().next();
         List<?> valuesOnP = (List)propertiesOnVertex.values().iterator().next();
         assertEquals(numProps, valuesOnP.size());
+        for (int i=0;i< numProps; i++) {
+          assertEquals(Integer.toString(i), valuesOnP.get(i).toString() );
```
Comment (mbrukman) [inline janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java]: Please delete extra space before closing parenthesis.

File: janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java [hunk 2]
```diff
@@ -70,12 +70,18 @@ public void testReadWideVertexWithManyProperties() throws Exception {
...
+        }
         Graph g = getGraph();
         GraphTraversalSource t = g.traversal(GraphTraversalSource.computer(SparkGraphComputer.class));
         assertEquals(numV, (long) t.V().count().next());
         propertiesOnVertex = t.V().valueMap().next();
         valuesOnP = (List)propertiesOnVertex.values().iterator().next();
         assertEquals(numProps, valuesOnP.size());
+        for (int i=0;i< numProps; i++) {
+          assertEquals(Integer.toString(i), valuesOnP.get(i).toString() );
```
Comment (mbrukman) [inline janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java]: Please delete extra space before closing parenthesis.

File: janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java [hunk 3]
```diff
@@ -70,12 +70,18 @@ public void testReadWideVertexWithManyProperties() throws Exception {
         Map<String, Object> propertiesOnVertex = graph.traversal().V().valueMap().next();
         List<?> valuesOnP = (List)propertiesOnVertex.values().iterator().next();
         assertEquals(numProps, valuesOnP.size());
+        for (int i=0;i< numProps; i++) {
```
Comment (mbrukman) [inline janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java]: For consistency and readability, please format with spaces around `=` and `<` and space after `;`:

```java
for (int i = 0; i < numProps; i++) {
```

File: janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java [hunk 4]
```diff
@@ -70,12 +70,18 @@ public void testReadWideVertexWithManyProperties() throws Exception {
...
+          assertEquals(Integer.toString(i), valuesOnP.get(i).toString() );
+        }
         Graph g = getGraph();
         GraphTraversalSource t = g.traversal(GraphTraversalSource.computer(SparkGraphComputer.class));
         assertEquals(numV, (long) t.V().count().next());
         propertiesOnVertex = t.V().valueMap().next();
         valuesOnP = (List)propertiesOnVertex.values().iterator().next();
         assertEquals(numProps, valuesOnP.size());
+        for (int i=0;i< numProps; i++) {
```
Comment (mbrukman) [inline janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java]: For consistency and readability, please format with spaces around `=` and `<` and space after `;`:

```java
for (int i = 0; i < numProps; i++) {
```

File: janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java (hunk 3 above)
Comment (nidmgh) [reply janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java]: thanks for the comment. I will fix it

File: janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java (hunk 1 above)
Comment (nidmgh) [reply janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java]: I will fix it too. thx

**Comment by jerryjch** [comment]
All review comments have been addressed.

**Pull request merged**